src/aircode.ts

```diff
@@ -755,6 +777,10 @@ export class AirCode implements vscode.FileSystemProvider {
     }
 
     copy?(source: vscode.Uri, destination: vscode.Uri, options: { readonly overwrite: boolean; }): void | Thenable<void> {
+        const sourcePath = source.path;
+        const destinationPath = destination.path;
+
+        return this.sendCommand(source, Command.CopyFile.from(sourcePath, destinationPath));
```

Comment:
Was this unimplemented before?

Reply:
Yeah! Documentation said this was optional (I thought maybe it would do a read/write instead), but it didn't work without it.